buildkite/test-front-end.dockerfile

```diff
@@ -1,7 +1,11 @@
 ARG GIT_ID="UNKNOWN"
 FROM mrcide/hint-shared-build-env:$GIT_ID
 
+ARG CODECOV_TOKEN
+ENV CODECOV_TOKEN=$CODECOV_TOKEN
+
 CMD npm run lint --prefix=app/static -- --quiet && \
-    npm test --prefix=app/static
-   # TODO codecov -f app/static/coverage/*.json && \
-   # TODO codecov -f app/static/coverage/integration/*.json
+    npm test --prefix=app/static && \
+    npm run integration-test-docker --prefix=app/static && \
```

Comment:
Also noticed we weren't running the front-end integration tests :see_no_evil: Added a new npm task that uses it's own jest config specifically for running inside docker, where the app is available at `http://hint:8080` instead of localhost